src/IRac.cpp

```diff
@@ -1250,6 +1254,37 @@ void IRac::hitachi424(IRHitachiAc424 *ac,
 }
 #endif  // SEND_HITACHI_AC424
 
+#if SEND_KELON
+/// Send a Kelon A/C message with the supplied settings.
+/// @param[in, out] ac A Ptr to an IRKelonAc object to use.
+/// @param[in] mode The operation mode setting.
+/// @param[in] degrees The temperature setting in degrees.
+/// @param[in] fan The speed setting for the fan.
+/// @param[in] sleep Run the device in sleep/quiet mode.
+/// @param[in] superCool Run the device in Super cooling mode.
+/// @param[in] dryGrade The dehumidification intensity grade
+/// @param[in] togglePower Whether to toggle the unit's power
+/// @param[in] toggleSwing Whether to toggle the swing setting
+void IRac::kelon(IRKelonAc *ac,
+                 const stdAc::opmode_t mode, const float degrees,
+                 const stdAc::fanspeed_t fan, const bool sleep,
+                 const bool superCool, const int8_t dryGrade,
+                 const bool togglePower = false, const bool toggleSwing = false) {
```

Comment:
Very minor style nitpick: Can you please keep the order of the settings parameters the same/similar to the other similar functions.
e.g. `power` first, then `mode`. `sleep` at/towards the end (after `clock`, which you don't use). Turbo (`superCool`) before `sleep`. swing before turbo, etc. Anything additional, put it near/next to the similar parameter.
i.e. `power, togglePower, ...`. Probably `mode, dryGrade` as it is related to dry mode.

This is just to try to keep things similar to others. Not a show stopper.